src/main/scala/afenton/bazel/bsp/BazelBspServer.scala

```diff
@@ -245,7 +245,7 @@ class BazelBspServer(
   def buildShutdown(params: Unit): IO[Unit] =
     for
       _ <- logger.info("build/shutdown")
-      _ <- IO.raiseError(new Exception("SHUTTING DOWN"))
+      // _ <- IO.raiseError(new Exception("SHUTTING DOWN"))
```

Comment:
can we remove the comment?

Why do we need this method? It looks like just a log call.

Reply:
Needs to shutdown server, changed in https://github.com/aishfenton/bazel-bsp/pull/6